libvast/include/vast/view.hpp

```diff
@@ -78,9 +78,9 @@ class pattern_view : detail::totally_ordered<pattern_view> {
 public:
   static pattern glob(std::string_view x);
 
-  explicit pattern_view(const pattern& x);
+  explicit pattern_view(const pattern& x, bool case_insensitive = false);
```

Comment:
True, it would be a simple removal of this tautology. I implemented a simple `case_insensitive` getter method in https://github.com/tenzir/vast/pull/2951/commits/9f8007bf38c4b350f75fd061673f81b6cd53652c